packages/react/src/Card/Card.module.css

```diff
@@ -92,6 +92,10 @@
   border-radius: var(--brand-borderRadius-xlarge);
 }
 
+.Card__outer:has(.Card--variant-minimal) {
+  border-radius: var(--brand-borderRadius-small);
```

Comment:
Was this supposed to be medium instead of small? I recall some conversation about this on the call on using medium. Personal opinion, but I think it looks better as small as possible. cc. @simmonsjenna.

Reply:
Yeah that works me for personally, thanks @danielguillan